### DISCUSSION TITLE
🔗 Close button for Off Canvas Widget (Duplicate of #27474)

### DISCUSSION THREAD

**Discussion by tamelander** [elementor/elementor#27992] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

On a narrow mobile screen, the off canvas can display over the whole screen and "clicking outside" is not possible.

### Describe the solution you'd like.

A close button.

### Describe alternatives you've considered.

I've tried to call the Elementor off canvas javascript with a custom javascript and an custom close button in the off canvas content, but it works only once. The off canvas div gets a style style=display:none; assigned when I close it with my custom close button. And that style stays so the off canvas can't be displayed again.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
You can use any element with a Link field to close the Off-Canvas. In the link field select the Dynamic Tag > Off-Canvas, and choose the option to "Close". Then you can position it anywhere inside the Off-Canvas.

Here's an example:


https://github.com/user-attachments/assets/4bb00ade-7177-4b3f-9098-cd1cbcf404a3



In this case I used an Icon widget, and changed its position to Absolute, aligned to the Top Right corner.

You can learn more about how to use the Dynamic Tags to control the Off-Canvas widget in this article:
https://elementor.com/help/off-canvas-widget/

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #27474

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
